config/Version.public.xcconfig

```diff
@@ -1,4 +1,4 @@
-VERSION_SHORT=18.7
+VERSION_SHORT=18.8
 
 // Public long version example: VERSION_LONG=9.9.0.0
-VERSION_LONG=18.7.0.2
+VERSION_LONG=18.8.0.1
```

Comment:
The `complete_code_freeze` lane failed on my end because of this error:

    ```
    /Users/gio/Developer/a8c/wpios/vendor/bundle/ruby/2.7.0/gems/fastlane-2.195.0/fastlane_core/lib/fastlane_core/ui/interface.
    remote: This repository moved. Please use the new location:
    remote:   git@github.com:wordpress-mobile/WordPress-iOS.git
    To github.com:wordpress-mobile/wordpress-ios
       ce978a3267..4251fe5a6a  release/18.8 -> release/18.8
     ! [rejected]              16.6 -> 16.6 (already exists)
    error: failed to push some refs to 'github.com:wordpress-mobile/wordpress-ios'
    ```

I pushed manually, then run the remaining command in the lane: starting  the CI release build.

Unfortunately, I used my shell history completion and didn't realize I called the lane to trigger a production release, not a beta.

I noticed that only when looking at the releases in GitHub and seeing `18.8` instead of the expected `18.8.0.0`. By then, a build had already been submitted to App Store Connect for WordPress, making it necessary to bump the number here.

Reply:
Ouch 🧸 

I'm not sure I understand why you got the "repository moved" error in the first place though? 🤔  Seems unrelated to your mistake of using a production lane?! Strange 🤔